src/app/app.module.ts

```diff
@@ -3,14 +3,20 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HomePageModule } from './modules/home-page/home-page.module';
+import { PointsShopPageModule } from './modules/pointsshop-page/pointsshop-page.module';
+import { SharedModule } from './modules/shared/shared.module';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+	SharedModule,
+	HomePageModule,
+	PointsShopPageModule
   ],
   providers: [],
   bootstrap: [AppComponent]
```

Comment:
If you don't use any section of the config, you can feel free to remove it!
```suggestion
  ],
  bootstrap: [AppComponent]
```